README.md

```diff
@@ -14,7 +14,7 @@ Bluejay aims to be an open source successor to BLHeli_S adding several improveme
 ## Current Features
 
 - Digital signal protocol: DShot 150, 300 and 600
-- Bidirectional DShot: RPM telemetry
+- Bidirectional DShot: RPM telemetry and [extended DSHOT telemetry](https://github.com/bird-sanctuary/extended-dshot-telemetry)
```

Comment:
For BB21 snd BB51 speeds are now 300, and 600, for DSHOT, BB51 supported 150, and 300

Reply:
Fixed.